packages/kbn-react-field/src/field_icon/field_icon.tsx

```diff
@@ -69,6 +70,7 @@ export const typeToEuiIconMap: Partial<Record<string, EuiTokenProps>> = {
   _source: { iconType: 'editorCodeBlock', color: 'gray' },
   point: { iconType: 'tokenShape' }, // there is no separate icon for `point` yet
   shape: { iconType: 'tokenShape' },
+  sparse_vector: { iconType: 'tokenDenseVector' },
```

Comment:
Same question here, do we want to use tokenDenseVector or should we create a new token for sparse vector?

Reply:
@andreadelrio do we have an icon we can use or add to eui?

Reply:
Andrea is already pinged in https://github.com/elastic/eui/issues/7257#issuecomment-1749376019. 😉  My plan is to open a separate PR when that's ready.